tools/bin/commands/crc

```diff
@@ -47,6 +45,11 @@ EOT
 crc::run() {
     source "$(basedir)/commands/util/openshift_funcs"
     source "$(basedir)/commands/util/camel_k_funcs"
+    source "$(basedir)/commands/util/operator_funcs"
+
+    OPERATOR_BINARY="$(basedir)/commands/binaries/syndesis"
```

Comment:
```suggestion
    OPERATOR_BINARY="$HOME/.syndesis/bin/syndesis"
```

Perhaps download outside of the source tree?

Reply:
why? it is included in .gitignore

Reply:
This might be recursive: so it doesn't have to be in .gitignore :) Also might be better not to tie it to a single git checkout, this is different for shell scripts that are cross platform.

Reply:
I still don't buy that idea, I don't see how this improves anything. We have the syndesis binary under `tools/bin`, why having this in a different place.

BUT! I will do it